syncmaster/worker/spark.py

```diff
@@ -51,7 +52,12 @@ def get_packages(db_type: str) -> list[str]:
         import pyspark
 
         spark_version = pyspark.__version__
-        return SparkS3.get_packages(spark_version=spark_version)
+        return SparkS3.get_packages(spark_version=spark_version) + Excel.get_packages(spark_version=spark_version)
+    if db_type == "hdfs":
```

Comment:
s3 too can contain Excel files

Reply:
Can you elaborate on that? Isn't `Excel.get_packages(spark_version=spark_version)` enough for S3?

Reply:
My bad, now I see you've added `Excel.get_packages` to S3 too